lib/client/Cargo.toml

```diff
@@ -31,9 +31,11 @@ solana-client = { workspace = true }
 solana-rpc = { workspace = true }
 solana-sdk = { workspace = true }
 solana-address-lookup-table-program = { workspace = true }
-mango-feeds-connector = "0.1.1"
+# groovie/MAN-68-mango-feeds-dependencies but with solana 1.16.1
+mango-feeds-connector = { git = "https://github.com/blockworks-foundation/mango-feeds.git", branch = "update-solana-1-16-anchor-28", features = ["solana-1-16"] }
```

Comment:
guess _connector_ dependency need to stay on a branch until this one gets merged - wdyt?

Reply:
alright, but let's movev it to a nicer dependency immediately when mango-feeds gets merged - we can't publish mango-v4-client to cargo with a dep like this, I think